src/layouting/transformation.jl

```diff
@@ -298,6 +298,14 @@ function apply_transform(f, r::Rect)
     ma_t = apply_transform(f, ma)
     Rect(Vec(mi_t), Vec(ma_t .- mi_t))
 end
+function apply_transform(f::PointTrans, r::Rect)
+    mi = minimum(r)
+    ma = maximum(r)
+    mi_t = apply_transform(f, Point(mi))
+    ma_t = apply_transform(f, Point(ma))
+    Rect(Vec(mi_t), Vec(ma_t .- mi_t))
+end
```

Comment:
@SimonDanisch Would it make sense to change `apply_transform(f, r::Rect)` to operate on `Point` types as well?
https://github.com/JuliaPlots/Makie.jl/blob/284222754f4106ffb6829d39b612e5011e297db7/src/layouting/transformation.jl#L301-L307
or even change the definition of `minimum(::Rect)`/`maximum(::Rect)` to give a `Point`? I don't quite understand enough about how they're used to make the change, but if we do that then we can delete this method.